### PR TITLE
Upgrade to Decidim v0.23

### PR DESCRIPTION
**Pull request by LauraJaimeC**
Changelog: https://github.com/decidim/decidim/blob/release/0.23-stable/CHANGELOG.md

- Update Decidim to v0.23
- Changes in geocoder configuration

!!!!!
This PR depends of https://github.com/CodiTramuntana/decidim-erc-crm_authenticable/pull/17
It is necessary to change the Gemfile when this PR will merge.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ source 'https://rubygems.org'
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: 'release/0.21-stable' }
-TERM_CUSTOMIZER_VERSION = { git: 'https://github.com/mainio/decidim-module-term_customizer', branch: '0.21-stable'}
+DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: 'release/0.23-stable' }
```
Comment (tramuntanal) [inline Gemfile:7]: We have many fixes for 0.23 in our fork, maybe it will be better to use that repo, same branch

**Review by tramuntanal — CHANGES_REQUESTED** [review]
I think it's better to use our fork for this version @LauraJaimeC 

Also, we must remember this from the changelog:
> Debates and Comments have been added to the global search and need to be indexed, otherwise all previous content won't be available as search results. You should run this in a Rails console at your server or create a migration to do it.
>
> Please be aware that it could take a while if your database has a lot of content.
> ```
>  Decidim::Comments::Comment.find_each(&:try_update_index_for_search_resource)
>  Decidim::Debates::Debate.find_each(&:try_update_index_for_search_resource)
> ```

**Comment by LauraJaimeC** [comment]
> I think it's better to use our fork for this version @LauraJaimeC
> 
> Also, we must remember this from the changelog:
> 
> > Debates and Comments have been added to the global search and need to be indexed, otherwise all previous content won't be available as search results. You should run this in a Rails console at your server or create a migration to do it.
> > Please be aware that it could take a while if your database has a lot of content.
> > ```
> >  Decidim::Comments::Comment.find_each(&:try_update_index_for_search_resource)
> >  Decidim::Debates::Debate.find_each(&:try_update_index_for_search_resource)
> > ```

This content of changelog it's already in upgrades doc

**Review by tramuntanal — CHANGES_REQUESTED** [review]
It seems that the schema.rb has not been updated

**Comment by LauraJaimeC** [comment]
> It seems that the schema.rb has not been updated

I have tried to update and it won't let me. I've deleted the schema and I run migrates and it doesn't change.

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**